konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/pipeline/graph/MergeStep.java

```diff
@@ -33,13 +35,14 @@
  * @author Alex Black
  */
 @AllArgsConstructor
-@EqualsAndHashCode(callSuper = true)
+@Data
```

Comment:
Adding the `@Data` is ok but we still need the `@EqualsAndHashCode(callSuper = true)` here.
If you look at the build warnings, you'll find this:
```
Warning:(38, 1) java: Generating equals/hashCode implementation but without a call to superclass, even though this class does not extend java.lang.Object. If this is intentional, add '@EqualsAndHashCode(callSuper=false)' to your type.
```